### PR TITLE
[T4A5][W11-B4] Charlton Lim 

### PR DESCRIPTION
**Pull request by cadmusthefounder**

### CODE REVIEW

**Comment by cadmusthefounder** [comment]
Ready for review.

File: src/seedu/addressbook/data/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/Tagging.java:7]: Please include a header comment to indicate what this interface/class/method does?

File: src/seedu/addressbook/data/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+public class Tagging {
+    public static ArrayList<Tagging> allTagChanges = new ArrayList<Tagging>();
```
Comment (okkhoy) [inline src/seedu/addressbook/data/Tagging.java:9]: recollect the discussion during tutorial on why this is not the right place to maintain the tagging list.
please check and differentiate the 2 diagrams in the activity description for better understanding

**Comment by okkhoy** [comment]
@cadmusthefounder 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**